src/illuminatio/__init__.py

```diff
@@ -1,10 +1,13 @@
 # -*- coding: utf-8 -*-
+"""
+mandatory file for this python module
+"""
 from pkg_resources import get_distribution, DistributionNotFound
 
 try:
     # Change here if project is renamed and does not equal the package name
-    dist_name = __name__
-    __version__ = get_distribution(dist_name).version
+    DIST_NAME = __name__
+    __version__ = get_distribution(DIST_NAME).version
```

Comment:
Any reason we can't use `__version__ = get_distribution(__name__).version` ? I don't see that `DIST_NAME` is used anywhere else.